### DISCUSSION TITLE
Successfully installed ExpensesBundle, but Kimai tells "There are no plugins"

### DISCUSSION THREAD

**Discussion by kernie** [kimai/Paid-plugins#78] [Questions & Answers] [answered]
Hi,
as title says I did all the installation as described here: https://www.kimai.org/store/expenses-bundle.html

Got the console message "Congratulations! Plugin was successful installed: ExpensesBundle" but going to Kimai website and checking plugins, it says: "Sie haben noch kein Plugin installiert." (no plugins installed).

I have:
Kimai 2.0.24
ExpensesBundle: 2.2.1
Php 8.1.18
Apache 2.4.57

Any ideas?

**Comment by kernie** [comment, marked as answer]
OK. Sorry for false alarm.

After restart of the apache webserver "service php8.1-fpm restart && service apache2 restart" everything worked.

**Reply by kevinpapst** [reply]
That must be the opcache, which is disabled on CLI.
Thanks for sharing the solution 👍
